ablySpec/RealtimeClientConnection.swift

```diff
@@ -1302,6 +1302,54 @@ class RealtimeClientConnection: QuickSpec {
 
             }
 
+            // RTN15
+            context("connection failures once CONNECTED") {
+
+                // RTN15d
+                pending("should recover from disconnection and messages delivered once the connection is resumed") {
```

Comment:
You right, I noticed [that](https://github.com/ably/ably-ios/pull/244#issuecomment-187715273). I will change the #244.